mempool/cache.go

```diff
@@ -88,8 +86,7 @@ func (c *LRUTxCache) Push(tx types.Tx) bool {
 	return true
 }
 
-func (c *LRUTxCache) Remove(tx types.Tx) {
-	key := tx.Key()
+func (c *LRUTxCache) Remove(key types.TxKey) {
 	c.RemoveTxByKey(key)
 }
```

Comment:
[question] `Remove` is now redundant with `RemoveTxByKey`. Should we remove one of them?

Reply:
would that no be api breaking?

Reply:
good point, let's keep both if they were already part of the public API already